src/reducers/fetchPlayersReducer.js

```diff
@@ -0,0 +1,11 @@
+export default (state = [], action) => {
+    switch (action.type) {
+        case 'FETCH_PLAYERS':
+            return action.payload;
+        default:
+            return state;
+    }
+};
```

Comment:
Instead of calling the file `fetchPlayersReducer` consider calling it `playersListReducer`

Also, I prefer `GET_PLAYERS_LIST` over `FLETCH_PLAYERS`